### PR TITLE
Add available experiences for player

### PR DESCRIPTION
**Pull request by henrymcarvajal-galilei**

### CODE REVIEW

File: web/src/utils/verbalizePassedDays.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
+// Credits to https://gist.github.com/flangofas/714f401b63a1c3d84aaa
+const convertMilliseconds = (milliseconds, format) => {
+    let days, hours, minutes, seconds, total_hours, total_minutes, total_seconds;
```
Comment (andymotagl) [inline web/src/utils/verbalizePassedDays.js:3]: Can these be camelCase and constants?

File: web/src/App.css [hunk 2]
```diff
@@ -69,6 +69,34 @@
...
+  overflow: hidden;
+}
+
+.player-experiences__container__table-header {
+  position: sticky;
+  top: 0;
+  z-index: 1;
+}
+
+.player-experiences__container__table-body {
```
Comment (andymotagl) [inline web/src/App.css:90]: Can we save some spaces with just some `-` instead of `__` ?

**Review by andymotagl — COMMENTED** [review]
Looks good to me! Please share the design on dark to match the current design then this will be shipped

File: web/src/App.css (hunk 2 above)
Comment (henrymcarvajal-galilei) [reply web/src/App.css:90]: Using BEM notation, but these styles were removed!

File: web/src/utils/verbalizePassedDays.js (hunk 1 above)
Comment (henrymcarvajal-galilei) [reply web/src/utils/verbalizePassedDays.js:3]: Fixed

File: web/src/client/questionary/Questionary.js [hunk 3]
```diff
@@ -1,41 +1,44 @@
...
+import { useSelector } from 'react-redux';
+import { useLocation } from 'react-router-dom';
+import axios from 'axios';
 
-import "../../assets/css/client.css"
+// Internal Utilities and Constants
+import { BACKEND_URL } from '../../utils/constants.js';
 
-//context
+// Context
```
Comment (andymotagl) [inline web/src/client/questionary/Questionary.js:10]: Remove?

File: web/src/utils/verbalizePassedDays.js (hunk 1 above)
Comment (andymotagl) [reply web/src/utils/verbalizePassedDays.js:3]: I still see snake case and non constant variables

File: web/src/views/experience/ExperienceDetail.js [hunk 4]
```diff
@@ -1,66 +1,86 @@
...
 import { useToken } from '../../context/TokenContext';
-import BackToExperiencesButton from './BackToExperiencesButton';
+
+// API Hooks
 import { useFetchExperience } from '../../api/ExperiencesAPI';
 
+// Components
+import BackToExperiencesButton from './BackToExperiencesButton';
+
+// Assets (Fonts, Images, CSS, SFX)
```
Comment (andymotagl) [inline web/src/views/experience/ExperienceDetail.js:15]: For me the imports speak by themselves as we have it

File: web/src/utils/verbalizePassedDays.js (hunk 1 above)
Comment (henrymcarvajal-galilei) [reply web/src/utils/verbalizePassedDays.js]: Solved!

File: web/src/client/experiences/ListSection.js [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+    return (
+        <div>
+            <div className="section-text">
+                <h1 className="section-index">Sección {section}</h1>
+                <h2 className="section-title">{title}</h2>
+            </div>
+            <div>{verbalizePassedDays(experience.created_at)}</div>
+            <div className="section-icon">
+                <picture className="section-img">
+                    <img alt="" src={sectionIcon} onClick={() => navigate(`/client/experience/${experience.experience_id}`)}/>
```
[the remark this answers is not in the document]
Comment (henrymcarvajal-galilei) [reply web/src/client/experiences/ListSection.js]: Solved!

File: web/src/client/questionary/Questionary.js (hunk 3 above)
Comment (henrymcarvajal-galilei) [reply web/src/client/questionary/Questionary.js:9]: Fixed!

File: web/src/views/experience/ExperienceDetail.js (hunk 4 above)
Comment (henrymcarvajal-galilei) [reply web/src/views/experience/ExperienceDetail.js:15]: Cleaned!

File: web/src/client/experiences/GameSection.js [hunk 6]
```diff
@@ -0,0 +1,28 @@
+// External Libraries
```
[the remark this answers is not in the document]
Comment (henrymcarvajal-galilei) [reply web/src/client/experiences/GameSection.js]: Fixed!

**Review by andymotagl — APPROVED** [review]

**Pull request merged**